front/components/groups/GroupsList.tsx

```diff
@@ -84,7 +84,6 @@ export type GroupsListProps = {
 };
 
 export function GroupsList({
-  searchTerm,
```

Comment:
It was not included in design and I think it's okay that you cannot search anymore?